inc/telemetry.class.php

```diff
@@ -114,14 +118,16 @@ static public function grabDbInfos() {
 
       $dbinfos = $DB->getInfo();
 
-      $size_res = $DB->query("SELECT ROUND(SUM(data_length + index_length) / 1024 / 1024, 1) dbsize
+      // Specifics informations (DB size, DB motor engine)
+      $size_res = $DB->query("SELECT ROUND(SUM(data_length + index_length) / 1024 / 1024, 1) dbsize, engine
          FROM information_schema.tables WHERE table_schema='" . $DB->dbdefault ."'");
       $size_res = $DB->fetch_assoc($size_res);
 
       $db = [
          'engine'    => $dbinfos['Server Software'],
          'version'   => $dbinfos['Server Version'],
          'size'      => $size_res['dbsize'],
+         'motor'     => $size_res['engine'],
```

Comment:
Sounds strange to me.
Anyways, since defaults become innodb, I'm really not sure this information is really relevant (engine is version specific). Also, innodb/myisam can be mixed.

Reply:
i personally asked for this one, the answer may be the majority of tables in a engine

Reply:
what is the poit of getting that? what will other than mysql databass will return?

Reply:
empty string ?

Reply:
for point, tracking the migration is done well on existing instance, knowing if we need to insist on this point, no ?